azurerm/internal/services/authorization/tests/resource_arm_role_definition_test.go

```diff
@@ -170,6 +170,26 @@ func TestAccAzureRMRoleDefinition_emptyName(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMRoleDefinition_managementGroup(t *testing.T) {
```

Comment:
@katbyte as per my understanding this should test the changes, if TF_ACC is set.